logos/src/logos/main.py

```
@@ -0,0 +1,280 @@
+import json
+import os
+import traceback
+
+import grpc
+from fastapi import FastAPI, Request
+from fastapi.middleware.cors import CORSMiddleware
+
+from grpclocal import model_pb2_grpc
+from grpclocal.grpc_server import LogosServicer
+from logos.dbutils.dbmanager import DBManager
+from logos.dbutils.dbrequest import *
```

Ruff (0.11.9)

`from logos.dbutils.dbrequest import *` used; unable to detect undefined names

(F403)